src/Components/Popup.js

```diff
@@ -27,11 +26,6 @@ export default class Popup {
   };
 
   setEventListeners() {
-    // sets event listeners that adds a click event listener to the close icon
-    // of the popup
-    // The modal window should also close when users click on the shaded
-    // area around the form.
-
```

Comment:
`    document.addEventListener("click", this._handleClickOutside);
`


Please, don’t add listeners to `document` or `window` if you don’t remove them because the listeners always watch any clicks on the page and make checks. That’s a waste of resources.